modules/localization/msf/local_tool/local_visualization/engine/visualization_engine.h

```diff
@@ -103,6 +101,49 @@ struct LocalizatonInfo {
   bool is_has_std;
 };
 
+/**
+ * @struct VisualMapParam
+ * @brief The data structure to store parameters of a map
+ */
+struct VisualMapParam {
+  VisualMapParam() {
+    map_node_size_x = 0;
```

Comment:
Just init on declaration?

Reply:
Yes. The variables will be set by the following function.